scripts/common/kubernetes.sh

```diff
@@ -188,10 +188,16 @@ function spinner_kubernetes_api_stable() {
 }
 
 function kubernetes_drain() {
-    kubectl drain "$1" \
+    local evict=""
+    # Check whether existing tools support bypassing eviction protection
```

Comment:
Would it hurt to switch the order for upgrades to first install the new kubectl version?

Reply:
Not sure if there's a specific reason to perform drain and `kubeadm upgrade` with the old version.

Reply:
https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#upgrade-the-first-control-plane-node actually specifically suggests upgrading kubeadm before using it to upgrade the cluster.

Reply:
We do upgrade kubeadm binary before running `kubeadm upgrade` but we don't upgrade kubectl until after.

Reply:
Ok. I must've missed that part.